roles/fio-devel/vars/main.yml

```diff
@@ -0,0 +1,5 @@
+# fio tag to checkout
```

Comment:
Instead of putting stuff on \<role\>/vars/main.yml, you could put them on \<role\>/default/main.yml so that they can be overriden in the playbook.

See [role directory structure](https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse_roles.html#role-directory-structure) and [variable precedence]( https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#understanding-variable-precedence)

Reply:
Oh thanks. I had that wrong!